packages/web-new/tests/test.spec.js

```diff
@@ -0,0 +1,2259 @@
+        await expect(exampleEditor).toHaveAttribute('data-mode-id', "json")
+        await expect(exampleEditor).toHaveClass("editor active")
+
+        const AsyncAPIView = page.locator('#async-api-view')
+        const consoleError = page.locator('#console-error')
+        const AsyncAPITab = page.locator("#async-api-tab")
+
+        await expect(AsyncAPIView).toHaveClass("console-view async-api-view hidden")
+        await expect(consoleError).toHaveClass("console-view console-error hidden")
+        await expect(AsyncAPITab).toBeChecked({ checked: false })
+
+        await AsyncAPITab.click()
+
+        await expect(AsyncAPITab).toBeChecked({ checked: true })
+        await expect(AsyncAPIView).toHaveClass("console-view async-api-view")
+        await expect(consoleError).toHaveClass("console-view console-error hidden")
+
+        const AsyncAPIEditor = page.locator('#async-api-container')
+
+        await expect(AsyncAPIEditor).toHaveAttribute('data-mode-id', "json")
+
+        const AsyncAPIJsonBtn = page.locator('#async-api-json')
+        const AsyncAPIYamlBtn = page.locator('#async-api-yaml')
+
+        await expect(AsyncAPIJsonBtn).toBeDisabled()
+
+        await AsyncAPIYamlBtn.click()
+
+        await expect(AsyncAPIEditor).toHaveAttribute('data-mode-id', "yaml")
+        await expect(AsyncAPIYamlBtn).toBeDisabled()
+
+        const AsyncAPIContainer = AsyncAPIEditor.getByRole('code').locator('div').filter({ hasText: 'asyncapi: 2.0.0' }).nth(4)
+        await expect(AsyncAPIContainer).toHaveText('asyncapi: 2.0.0')
+
+        // Start waiting for download before clicking.
+        const AsyncAPIDownload = page.locator('#async-api-download')
+        const downloadPromise = page.waitForEvent('download')
+        await AsyncAPIDownload.click()
+        const download = await downloadPromise
+        const expectedFilename = 'MyLampThing-AsyncAPI.yaml'
+        expect(download.suggestedFilename()).toBe(expectedFilename)
+    })
+    */
+})
+
+
+test.describe("AAS AID view functionality", () => {
+    test("Trying to open the AAS view with a TD with no protocols and closing it", async ({ page }) => {
+
+        const initialTab = page.locator("#tab").nth(0)
+        await expect(initialTab).toHaveAttribute('data-tab-id', "1")
+        await expect(initialTab).toHaveText("TDThing TemplateCloseCancel")
+        await expect(initialTab).toHaveClass("active")
+
+        const AASView = page.locator('#aas-view')
+        const consoleError = page.locator('#console-error')
+        const AASTab = page.locator("#aas-tab")
+
+        await expect(AASView).toHaveClass("console-view aas-view hidden")
+        await expect(consoleError).toHaveClass("console-view console-error hidden")
+        await expect(AASTab).toBeChecked({ checked: false })
+
+        await AASTab.click()
+
+        await expect(AASTab).toBeChecked({ checked: true })
+        await expect(AASView).toHaveClass("console-view aas-view hidden")
```

Comment:
This one is failing with 
```
Expected string: "console-view aas-view hidden"
Received string: "console-view aas-view"
```

Reply:
I think the expected string is wrong since you click on the tab thus make it not hidden anymore